.github/workflows/ci.yml

```diff
@@ -758,13 +758,15 @@ jobs:
         target:
           - platform: "linux"
             arch: "x86_64"
-            docker_image: "gcr.io/iree-oss/base@sha256:f26a5aa5f8d3705c6b80c71d04fafb360861f1907bdd1b1f5f19480b6192664e"
+            docker_image: "gcr.io/iree-oss/base-bleeding-edge@sha256:10a94f7edf0df2f3894038e0e434a93ea353460ed72240c226548025690a79e5"
             # Builds tools on the host and assumes the builder is Linux x86_64.
             build_script: "./build_tools/cmake/build_runtime.sh"
+            tracy_capture: "gs://iree-shared-files/tracy-capture-linux-x86_64-52b6af88.tgz"
           - platform: "linux"
             arch: "riscv_64"
             docker_image: "gcr.io/iree-oss/riscv@sha256:a7e7b03236324f92e7c96925add02cf0b37eca5b68af54f64049fc689f770915"
             build_script: "./build_tools/cmake/build_riscv.sh"
+            tracy_capture: "gs://iree-shared-files/tracy-capture-linux-x86_64-52b6af88.tgz"
```

Comment:
(commenting here for threading)

Can you share some information about where the collected trace files show up? Maybe link within some logs or post a screenshot in the PR description?